tools/Test/SmokeTest/RmCoreSmokeTests.ps1

```diff
@@ -37,83 +37,118 @@ $resourceGroupName = "azpssmokerg$randomValue"
+                                                                try {
+                                                                    Remove-AzStorageAccount -Name $storageAccountName -ResourceGroupName $resourceGroupName -Force -ErrorAction Stop
+                                                                    break
+                                                                }
+                                                                catch {
+                                                                    if (++$loopLimit -gt 30)
+                                                                    {
+                                                                        Write-Warning "Give up Az.Storage [Cleanup]"
+                                                                        break
+                                                                    } else {
+                                                                        Start-Sleep -Seconds 30
+                                                                        Write-Warning "Retry Az.Storage [Cleanup]"
+                                                                    }
+                                                                }
+                                                            } while ($true)
+                                                        }},
+    @{Name = "Az.Resources [Cleanup]";        Command = {
+                                                            $loopLimit = 0
+                                                            do {
+                                                                try {
+                                                                    Remove-AzResourceGroup -Name $resourceGroupName -Force -ErrorAction Stop
+                                                                    break
+                                                                }
+                                                                catch {
+                                                                    if (++$loopLimit -gt 30)
+                                                                    {
+                                                                        Write-Warning "Give up Az.Resources [Cleanup]"
+                                                                        break
+                                                                    } else {
+                                                                        Start-Sleep -Seconds 30
+                                                                        Write-Warning "Retry Az.Resources [Cleanup]"
+                                                                    }
+                                                                }
+                                                            } while ($true)
+                                                        }}
```

Comment:
```suggestion
    @{Name = "Az.Resources [Cleanup]";        Command = {
                                                            $loopLimit = 0
                                                            do {
                                                                try {
                                                                    Remove-AzResourceGroup -Name $resourceGroupName -Force -ErrorAction Stop
                                                                    break
                                                                }
                                                                catch {
                                                                    if (++$loopLimit -gt 30)
                                                                    {
                                                                        Write-Warning "Give up Az.Resources [Cleanup]"
                                                                        break
                                                                    } else {
                                                                        Start-Sleep -Seconds 30
                                                                        Write-Warning "Retry Az.Resources [Cleanup]"
                                                                    }
                                                                }
                                                            } while ($true)
                                                        }}
```
Remove resource group has no similar problem. The code doesn't need to be changed.

Reply:
Done